CONTRIBUTING.md

```diff
@@ -6,16 +6,23 @@ Before interacting with the Redwood community, please read and understand our [C
 
 **Table of Contents**
 
-- [Contributing](#Contributing)
-  - [Local Development](#Local-Development)
-    - [Copy and Watch](#Copy-and-Watch)
-      - [Specifying a RW_PATH](#Specifying-a-RW_PATH)
-    - [Local Package Registry Emulation](#Local-Package-Registry-Emulation)
-      - [Setting Up and Running a Local NPM Registry](#Setting-Up-and-Running-a-Local-NPM-Registry)
-      - [Publishing a Package](#Publishing-a-Package)
-      - [Installing Published Packages in Your Redwood App](#Installing-Published-Packages-in-Your-Redwood-App)
-  - [Running Your Redwood App's Local Server(s)](#Running-Your-Redwood-App's-Local-Server(s))
-  - [Releases](#Releases)
+- [Contributing](#contributing)
+  - [Local Development](#local-development)
+    - [Copy and Watch](#copy-and-watch)
+      - [Specifying a RW_PATH](#specifying-a-rwpath)
+    - [Local Package Registry Emulation](#local-package-registry-emulation)
+      - [Setting Up and Running a Local NPM Registry](#setting-up-and-running-a-local-npm-registry)
+      - [Publishing a Package](#publishing-a-package)
+      - [Installing Published Packages in Your Redwood App](#installing-published-packages-in-your-redwood-app)
+  - [Running Your Redwood App's Local Server(s)](#running-your-redwood-apps-local-servers)
+  - [Releases](#releases)
+    - [Troubleshooting](#troubleshooting)
+  - [Reference](#reference)
```

Comment:
Let's be more explicit about naming here. Maybe "CLI Reference: `redwood-tools`". Not sure what that does to the markdown link. But it improves findability.